tensorboard/data/server/run.rs

```diff
@@ -15,33 +15,30 @@ limitations under the License.
 
 //! Loader for a single run, with one or more event files.
 
-use log::warn;
+use log::{debug, warn};
 use std::collections::{BTreeMap, HashMap, HashSet};
 use std::fs::File;
 use std::io::BufReader;
 use std::path::PathBuf;
 use std::sync::RwLock;
+use std::time::{Duration, Instant};
 
 use crate::commit;
 use crate::data_compat::{EventValue, GraphDefValue, SummaryValue, TaggedRunMetadataValue};
 use crate::event_file::EventFileReader;
 use crate::proto::tensorboard as pb;
 use crate::reservoir::StageReservoir;
-use crate::types::{Step, Tag, WallTime};
+use crate::types::{Run, Step, Tag, WallTime};
 
 /// A loader to accumulate reservoir-sampled events in a single TensorBoard run.
 ///
 /// For now, a run loader always reads from [`File`]s on disk. In the future, this may be
 /// parameterized over a filesystem interface.
 #[derive(Debug)]
 pub struct RunLoader {
-    /// The earliest event `wall_time` seen in any event file in this run.
-    ///
-    /// This is `None` if and only if no events have been seen. Its value may decrease as new
-    /// events are read, but in practice this is expected to be the wall time of the first
-    /// `file_version` event in the first event file.
-    start_time: Option<WallTime>,
-
+    /// The run name associated with this loader. Used primarily for logging; the run name is
+    /// canonically defined by the map key under which this RunLoader is stored in LogdirLoader.
```

Comment:
nit: Since Rustdoc is Markdown, it’s [convention][1] to use backticks
around code atoms like `` `RunLoader` `` and `` `Commit` ``. You can
also write doc crosslinks like ``[`RunLoader`]``, but that’s more at
your discretion as to whether it’s useful.

(Throughout.)